src/server.ts

```diff
@@ -352,7 +353,34 @@ export class Server {
               const offersClaimed = offerSuccess
                 .offersClaimed()
                 // TODO: fix stellar-base types.
-                .map((offerClaimed: any) => {
+                .map((offerClaimedAtom: any) => {
+                  const offerClaimed = offerClaimedAtom.value();
+
+                  let sellerId: string = "";
+                  switch (offerClaimedAtom.switch()) {
+                    case xdr.ClaimAtomType.claimAtomTypeV0():
+                      sellerId = StrKey.encodeEd25519PublicKey(
+                        offerClaimed.sellerEd25519(),
+                      );
+                      break;
+                    case xdr.ClaimAtomType.claimAtomTypeOrderBook():
+                      sellerId = StrKey.encodeEd25519PublicKey(
+                        offerClaimed.sellerId().ed25519(),
+                      );
+                      break;
+                    // It shouldn't be possible for a claimed offer to have type
```

Comment:
Shouldn't this be an empty `case`? I guess we want to ignore this case but right now it will fall into `default` branch that throws an error.

Reply:
No, because a `ManageOffer` result with a `ClaimAtomType.claimAtomTypeLiquidityPool()` in it should never occur, so that would definitely be an error :+1: